chia/rpc/wallet_rpc_api.py

```diff
@@ -433,13 +433,17 @@ async def farm_block(self, request):
 
     async def get_wallets(self, request: Dict):
         assert self.service.wallet_state_manager is not None
-
+        include_data: bool = request.get("include_data", True)
         wallet_type: Optional[WalletType] = None
         if "type" in request:
             wallet_type = WalletType(request["type"])
 
         wallets: List[WalletInfo] = await self.service.wallet_state_manager.get_all_wallet_info_entries(wallet_type)
-
+        if not include_data:
+            result: List[WalletInfo] = []
+            for wallet in wallets:
+                result.append(WalletInfo(wallet.id, wallet.name, wallet.type, ""))
```

Comment:
do we still need this if we're moving coins to a separate store?

Reply:
No harm to have